mediainfo/src/main/java/io/github/anilbeesetti/nextlib/mediainfo/MediaInfoBuilder.kt

```diff
@@ -7,16 +7,17 @@ import android.util.Log
 import androidx.annotation.Keep
 import java.io.FileNotFoundException
 
-class MediaInfoBuilder(private val context: Context) {
+class MediaInfoBuilder(private val context: Context?) {
```

Comment:
Hey @TheBeastLT, Is there a reason for context to be null?

Reply:
Ahh, that's from my testing, since `context` is only needed when dealing with fs paths. Maybe `context` could be passed to the specific `from` method where it's needed instead of constructor, since you can invoke `from(String)` without it? Do you think that's resonable?

Reply:
That's a great suggestion. Can you remove the context parameter from the constructor and add as parameter to function where it needed.